<ui version="4.0" >
 <class>ScreenShotSaveDialog</class>
 <widget class="QDialog" name="ScreenShotSaveDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Page shooter</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >
    <normaloff>:/resources/images/poshuku.svg</normaloff>:/resources/images/poshuku.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QFormLayout" name="formLayout" >
     <property name="fieldGrowthPolicy" >
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="FormatCombobox_" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Quality:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QSlider" name="QualitySlider_" >
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="singleStep" >
          <number>5</number>
         </property>
         <property name="pageStep" >
          <number>20</number>
         </property>
         <property name="value" >
          <number>75</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox" >
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="value" >
          <number>75</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="PreviewBox_" >
     <property name="title" >
      <string>Preview</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QLabel" name="FileSizeLabel_" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="Scroller_" >
        <property name="widgetResizable" >
         <bool>true</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>612</width>
           <height>347</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox_" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>accepted()</signal>
   <receiver>ScreenShotSaveDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>528</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>rejected()</signal>
   <receiver>ScreenShotSaveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>596</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>QualitySlider_</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>206</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel" >
     <x>603</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QualitySlider_</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>604</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel" >
     <x>187</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
